### PR TITLE
Added recursive bypass feature

### PR DESCRIPTION
**Pull request by vkhv**

### CODE REVIEW

File: lib/svgo/coa.js [hunk 1]
```diff
@@ -131,6 +139,7 @@ module.exports = require('coa').Cmd()
         .end()
     .act(function(opts, args) {
 
+	    console.log(`opts - ${opts.folder}, args - ${args}`)
```
Comment (GreLI) [inline lib/svgo/coa.js]: Has it left from debugging? SVGO works on node 0.12 and higher, where is no support for template strings.

File: lib/svgo/coa.js [hunk 2]
```diff
@@ -53,6 +54,13 @@ module.exports = require('coa').Cmd()
         })
         .end()
     .opt()
+        .name('recursive').title('Input folder, optimize and rewrite all *.svg files')
+        .short('rf').long('recursive')
```
Comment (GreLI) [inline lib/svgo/coa.js]: Why `rf`?

Comment (vkhv) [inline lib/svgo/coa.js]: Now - r.
25d3f017a178f34429135771342006ac26f1248e

File: lib/svgo/coa.js (hunk 1 above)
Comment (vkhv) [inline lib/svgo/coa.js]: Yes, just debug, fix - 25d3f017a178f34429135771342006ac26f1248e

**Comment by GreLI** [comment]
Many things are wrong here.

**Pull request closed without merging**